src/response/errors.rs

```diff
@@ -47,8 +48,10 @@ struct ErrorMsg {
 //This needs refactored.
 impl fmt::Display for Error {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        debug!("error formatter");
```

Comment:
Think we can delete this one

Reply:
Oops. Was just trying to work out when they got called. Should have done a grep on debug!